openedx/features/course_experience/static/course_experience/js/spec/LatestUpdate_spec.js

```diff
@@ -0,0 +1,30 @@
+/* globals $, loadFixtures */
+
+import 'jquery.cookie';
+import { LatestUpdate } from '../LatestUpdate';
+
+
+describe('LatestUpdate tests', () => {
+  function createLatestUpdate() {
+    new LatestUpdate({ messageContainer: '.update-message', dismissButton: '.dismiss-message button' }); // eslint-disable-line no-new
+  }
+  describe('Test cookies', () => {
+    beforeEach(() => {
+      $.cookie('update-message', '');
+      loadFixtures('course_experience/fixtures/latest-update-fragment.html');
+    });
+
+    it('Test dismiss button', () => {
+      createLatestUpdate();
+      expect($('.update-message').attr('style')).toBe(undefined);
+      $('.dismiss-message button').click();
+      expect($('.update-message').attr('style')).toBe('display: none;');
+    });
+
+    it('Test cookie hides update', () => {
+      $.cookie('update-message', 'hide');
```

Comment:
I can add the second condition, where we void the cookie, recreate the message and then verify that it appears, but it seems as though it's not possible to get the expiration date of the cookie once it is set through JS (via [StackOverflow](https://stackoverflow.com/questions/1532193/reading-cookie-expiration-date)) so it isn't possible to check the expiration date, which I also find frustrating.